src/ray/raylet/scheduling/scheduler_resource_reporter.cc

```diff
@@ -205,5 +205,31 @@ void SchedulerResourceReporter::FillPendingActorInfo(
   }
 }
 
+void SchedulerResourceReporter::FillPendingActorInfo(rpc::ResourcesData &data) const {
```

Comment:
I might miss something here, why don't we need to set `resource_load` as well besides `resource_load_by_shape`.

Also `FillPendingActorInfo` is the same name as the existing method but seems they are doing completely different thing. Let's change the name to distinguish them.

Reply:
> I might miss something here, why don't we need to set `resource_load` as well besides `resource_load_by_shape`.
> 

This function is only called by the gcs node, which executes no tasks but only queueing them (which gcs actor scheduler is turned on). So it has nothing to do with `resource_load`, which needing `resource_load_by_shape` to record the pending info.

> Also `FillPendingActorInfo` is the same name as the existing method but seems they are doing completely different thing. Let's change the name to distinguish them.

Sure. How about `FillPendingActorCountByShape`?